packages/consumption/test/core/TestUtil.ts

```diff
@@ -462,15 +463,16 @@ export class TestUtil {
         });
     }
 
-    public static async uploadFile(from: AccountController, fileContent: CoreBuffer): Promise<File> {
+    public static async uploadFile(from: AccountController, parameters?: { fileContent?: CoreBuffer; expiredAt?: CoreDate; tags?: string[] }): Promise<File> {
         const params: ISendFileParameters = {
-            buffer: fileContent,
+            buffer: parameters?.fileContent ?? CoreBuffer.from(await fs.promises.readFile(`${__dirname}/../__assets__/test.txt`)),
```

Comment:
```suggestion
            buffer: parameters?.fileContent ?? CoreBuffer.from("test"),
```
don't forget to remove that test.txt file.